enhancements/proposed/20201230-containerd-default/container-default.md

```diff
@@ -0,0 +1,41 @@
+# CRI: Containerd by default
+
+* First proposed: 2020-11-08
+* Authors: Anders F Björklund (@afbjorklund)
+
+## Reviewer Priorities
+
+Please review this proposal with the following priorities:
+
+*   Does this fit with minikube's [principles](https://minikube.sigs.k8s.io/docs/concepts/principles/)?
+*   Are there other approaches to consider?
+*   Could the implementation be made simpler?
+*   Are there usability, reliability, or technical debt concerns?
+
+Please leave the above text in your proposal as instructions to the reader.
+
+## Summary
+
+Change default container runtime, from current "docker" to replacement "containerd".
+
+## Goals
+
+* Change from docker to containerd as default
+* Still allow building images with minikube
```

Comment:
add still allow "fast image building" (comparable to docker-env method)

Reply:
Well, docker-env only works with docker - but I guess you mean building on some other docker daemon and pushing.

Like how kind and microk8s are doing it, when there is no build support within the cluster. With a save|load (tar) pipe...